src/Snippet.js

```diff
@@ -4,17 +4,38 @@ const {
 } = require('./common');
 
 class Snippet {
-  constructor (id, ext) {
+  constructor (id, ext, owner, repo, ref, filePath) {
     this.id = id;
     this.ext = ext;
+    this.owner = owner;
```

Comment:
Wasn't the point of this change to make it so you don't need to store all of these individually? Why not use your `Repo` class?

Reply:
It needs to pass from the repo to the snippet - because the snippets and target files don't know anything about the repos.